frontend/src/__tests__/cypress/cypress/pages/projects.ts

```diff
@@ -29,6 +53,11 @@ class ProjectListPage {
     this.wait();
   }
 
+  visitError() {
+    cy.visit('/invalid-project-name');
+    this.waitError();
+  }
```

Comment:
Remove this and other related functions and tests.
This is already covered by `PageNotFound.cy.ts`.